examples/00-load/create-sphere.py

```diff
@@ -0,0 +1,239 @@
+
+"""
+# sphinx_gallery_thumbnail_number = 5
+
+# %%
+from __future__ import annotations
+
+import numpy as np
+
+import pyvista as pv
+
+# %%
+# Simple Sphere
+# ~~~~~~~~~~~~~
+# The quickest method to get a Sphere mesh is to use :func:`pyvista.Sphere`.
+
+mesh = pv.Sphere()
+mesh.plot(show_edges=True)
+
+# %%
+# This gives an :class:`pyvista.PolyData` mesh, i.e. a 2D surface. In this case, it is watertight
+# and encloses a volume.
+
+mesh
+
+# %%
+# The cells are triangles. For example, the first cell
+
+mesh.get_cell(0).type
+
+# %%
+# Structured quadrilateral mesh of Sphere
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The structure of the mesh can be important. Instead of a
+# triangulated mesh, it can be useful to have a structured
+# mesh that has an i-j-k ordering that allows for simplified
+# cell connectivity.
+#
+# The points are generated as a regular grid in spherical coordinates.
+# Here, we will used the convention that ``theta`` is the
+# azimuthal angle, similar to longitude on the globe.  ``phi`` is the
+# polar angle, similar to latitude on the globe.
+
+radius = 0.5
+ntheta = 9
+nphi = 12
+theta = np.linspace(0, 2 * np.pi, ntheta)
+phi = np.linspace(0, np.pi, nphi)
+
+r_, phi_, theta_ = np.meshgrid([radius], phi, theta, indexing='ij')
+x, y, z = pv.spherical_to_cartesian(r_, phi_, theta_)
+mesh = pv.StructuredGrid(x, y, z)
+
+# %%
+# The mesh has quadrilateral cells. The cells that look triangular
+# at the poles are actually degenerate quadrilaterals, i.e. two
+# points are coincident at the pole, as will be shown later.
+
+mesh.plot(show_edges=True)
+
+# %%
+# The mesh is of type :class:`pyvista.StructuredGrid`.
+
+mesh
+
+# %%
+# The first cell is at the top pole, and it is a Quadrilateral.
+
+cell = mesh.get_cell(0)
+cell.type
+
+# %%
+# The first cell has two degenerate points.
+
+cell.points
+
+# %%
+# The cells on either side of the 'seam' along the start and end of
+# the azimuthal component are not connected. Visualize this by plotting
+# the boundary edges of the mesh.
+
+boundaries = mesh.extract_feature_edges(
+    non_manifold_edges=False, feature_edges=False, manifold_edges=False
+)
+
+pl = pv.Plotter()
+pl.add_mesh(mesh, show_edges=True)
+pl.add_mesh(boundaries, line_width=10, color='red')
+pl.show()
+
+# %%
+# Generate quadrilateral mesh of Sphere
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This example shows how a more complicated mesh can be defined.
+#
+# In contrast to the example above, this example generates a mesh
+# that does not have degenerate points at the poles. Triangle cells
+# will be used at the poles.  First, regenerate the structured data.
+
+radius = 0.5
+ntheta = 9
+nphi = 12
+theta = np.linspace(0, 2 * np.pi, ntheta)
+phi = np.linspace(0, np.pi, nphi)
+
+# %%
+# We do not want duplicate points, so remove the duplicate in theta, which
+# results in 8 unique points in theta. Similarly, the poles at `phi=0` and
+# `phi=pi` will be handled separately to avoid duplicate points, which
```

Comment:
Format as literals instead of italics
```suggestion
# results in 8 unique points in theta. Similarly, the poles at ``phi=0`` and
# ``phi=pi`` will be handled separately to avoid duplicate points, which
```